base/loading.jl

```diff
@@ -699,7 +699,16 @@ function parse_cache_header(f::IO)
         push!(files, (String(read(f, n)), ntoh(read(f, Float64))))
     end
     @assert totbytes == 4 "header of cache file appears to be corrupt"
-    return modules, files
+    # read the list of modules that are required to be present during loading
+    required_modules = Dict{Symbol, UInt64}()
```

Comment:
consistency with https://github.com/JuliaLang/julia/blob/70fdbe83a3022363fadbedef344b88dc3f05ea88/base/loading.jl#L683

and

https://github.com/JuliaLang/julia/blob/70fdbe83a3022363fadbedef344b88dc3f05ea88/base/loading.jl#L683